@navikt/core/react/src/overlays/floating-menu/Menu.tsx

```diff
@@ -0,0 +1,1202 @@
+        (event) => {
+          if (isPointerMovingToSubmenu(event)) event.preventDefault();
+        },
+        [isPointerMovingToSubmenu],
+      )}
+      pointerGraceTimerRef={pointerGraceTimerRef}
+      onPointerGraceIntentChange={React.useCallback((intent) => {
+        pointerGraceIntentRef.current = intent;
+      }, [])}
+    >
+      <FocusScope
+        onMountAutoFocus={composeEventHandlers(onOpenAutoFocus, (event) => {
+          // when opening, explicitly focus the content area only and leave
+          // `onEntryFocus` in  control of focusing first item
+          event.preventDefault();
+          contentRef.current?.focus({ preventScroll: true });
+        })}
+        onUnmountAutoFocus={onCloseAutoFocus}
+      >
+        <DismissableLayer
+          asChild
+          disableOutsidePointerEvents={disableOutsidePointerEvents}
+          onEscapeKeyDown={onEscapeKeyDown}
+          onPointerDownOutside={onPointerDownOutside}
+          onFocusOutside={onFocusOutside}
+          onInteractOutside={onInteractOutside}
+          onDismiss={onDismiss}
+        >
+          <RovingFocus
+            asChild
+            descendants={descendants}
+            onEntryFocus={composeEventHandlers(onEntryFocus, (event) => {
+              // only focus first item when using keyboard
+              if (!rootContext.isUsingKeyboardRef.current)
+                event.preventDefault();
+            })}
+          >
+            <Floating.Content
+              role="menu"
+              aria-orientation="vertical"
+              data-state={getOpenState(context.open)}
+              data-aksel-menu-content=""
+              dir="ltr"
+              {...contentProps}
+              ref={composedRefs}
+              style={{ outline: "none", ...contentProps.style }}
+              onKeyDown={composeEventHandlers(
+                contentProps.onKeyDown,
+                (event) => {
+                  // submenu key events bubble through portals. We only care about keys in this menu.
+                  const target = event.target as HTMLElement;
+                  const isKeyDownInside =
+                    target.closest("[data-aksel-menu-content]") ===
+                    event.currentTarget;
+                  if (isKeyDownInside) {
+                    // menus should not be navigated using tab key so we prevent it
+                    if (event.key === "Tab") event.preventDefault();
+                  }
+
+                  // focus first/last item based on key pressed
+                  const content = contentRef.current;
+                  if (event.target !== content) return;
+                  if (!FIRST_LAST_KEYS.includes(event.key)) return;
+                  event.preventDefault();
+
+                  if (LAST_KEYS.includes(event.key)) {
+                    descendants.lastEnabled()?.node?.focus();
+                    return;
+                  }
+                  descendants.firstEnabled()?.node?.focus();
```

Comment:
We do since at initial open with pointer(-down) the content-frame has focus, not an item. This leads to current active item in descendants being -1 (undefined) and therefore not possible to use arrowDown/Up to change focus.